great_expectations/data_context/store/_store_backend.py

```diff
@@ -130,6 +130,32 @@ def set(self, key, value, **kwargs):
             logger.debug(str(e))
             raise StoreBackendError("ValueError while calling _set on store backend.")
 
+    def add(self, key, value, **kwargs):
```

Comment:
Left untyped as the rest of the file is untyped - I think we should definitely type this but it'll cause a bunch of cascading `mypy` issues. Can come back to this in a subsequent PR